### PR TITLE
fix: change ebtables rule to solve ping issues

### PR DESCRIPTION
**Pull request by Tejaswi-Goel**
Temporary fix - Modify dn_rules.sh to prevent ARP requests going into NFLOG filter to solve ping issues

Signed-off-by: Tejaswi Goel <Tejaswi_Goel@Dell.com>

### CODE REVIEW

**Comment by Tejaswi-Goel** [comment]
Pull-request updated, HEAD is now 347a6a99d9a8f4bce8a52e17e445519c56905bff

**Comment by Tejaswi-Goel** [comment]
Pull-request updated, HEAD is now e7b7472079425c3d961a978022e3b387d452c748

File: cfg/dn_rules.sh [hunk 1]
```diff
@@ -49,8 +49,6 @@ ebtables -t nat -A POSTROUTING -d Multicast --mark 0x1 -j DROP
 
 #Accept ARP request on eth0 (untagged/tagged interfaces) and don't copy it to NFLOG
 ebtables -A OUTPUT -p ARP -o eth0+ --arp-op Request -j ACCEPT
-#NFLOG to copy all ARP requests to netlink group 100
-ebtables -A OUTPUT -p ARP --arp-op Request --nflog-group 100 -j DROP
```
Comment (rakeshdatta) [inline cfg/dn_rules.sh]: Can u plz add a little detail about the root cause of this issue and how it is affecting the pings.

**Comment by jeff-yin** [comment]
Fixes https://github.com/open-switch/opx-nas-l3/issues/24

File: cfg/dn_rules.sh (hunk 1 above)
Comment (Tejaswi-Goel) [reply cfg/dn_rules.sh]: Added the comment on the issue link:  https://github.com/open-switch/opx-nas-l3/issues/24   #24

**Comment by jeff-yin** [comment]
@GarrickHe please provide review again

**Pull request merged**